californium-core/pom.xml

```diff
@@ -37,6 +37,35 @@
 	<build>
 		<plugins>
 			<plugin>
+				<groupId>org.apache.felix</groupId>
+				<artifactId>maven-bundle-plugin</artifactId>
+				<extensions>true</extensions>
+				<configuration>
+					<instructions>
+						<Export-Package>
+							org.eclipse.californium.core,
```

Comment:
for httpclient it should be httpclient-osgi ... for http-nio there is no OSGi version available.

To be honest I don't know if we could specify the import versions manually. Could be, would need to check the felix bundle documentation. On the other hand the question is if we are able to know the version exported, as it doesn't have to be the bundle version.